Doppler.BigQueryMicroservice.Test/EmailServiceTest.cs

```diff
@@ -0,0 +1,75 @@
+using AutoFixture;
+using Doppler.BigQueryMicroservice.Entitites.EmailSender;
+using Doppler.BigQueryMicroservice.Services;
+using Flurl.Http.Testing;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.Options;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Doppler.BigQueryMicroservice
+{
+    /// <summary>
+    /// Unit test for email service
+    /// </summary>
+    public class EmailServiceTest : IClassFixture<WebApplicationFactory<Startup>>
```

Comment:
I think that you do not need to implement `IClassFixture<WebApplicationFactory<Startup>>` in this case, I am missing something?

```suggestion
    public class EmailServiceTest
```

Reply:
esto sigue pendiente?